platform/pulp/tasking/tasks.py

```diff
@@ -64,6 +64,12 @@ def _queue_reserved_task(name, inner_task_id, resource_id, inner_args, inner_kwa
                           will ensure that no other tasks that want that same reservation will run
                           concurrently with yours.
     :type  resource_id:   basestring
+    :param inner_args:    The positional arguments to pass on to the task.
```

Comment:
Can these docstrings be converted to Napoleon per [PUP2](https://github.com/pulp/pups/blob/master/pup-0002.md)? This can be done as a separate PR, but I think it would be good. Here is a regex to help with the conversation:  https://pulp.plan.io/issues/2347?pn=1#note-14 Thank you for anything you do in this area.